statefulset-template/form-ui-schema.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": {
+    "ui:title": "Name",
+    "ui:help": "The name of the generated StatefulSet"
+  },
+  "namespace": {
+    "ui:title": "Namespace",
+    "ui:help": "The target namespace for the generated StatefulSet",
+    "ui:widget": "namespaceSelection"
+  },
+  "image": {
+    "ui:title": "Image",
+    "ui:help": "Image for the container"
+  },
+  "labels": {
+    "ui:title": "Labels",
+    "ui:help": "Labels for linking objects"
```

Comment:
specify a default input type as placeholder

Reply:
Where?

Reply:
in the `ui:help` section you can add something like `ie. pvc: nginx-app`